packages/localize/src/localize.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+/// <reference path="../localize.d.ts" />
+
+/**
+ * Localize a template literal string.
+ *
+ * This implementation of the global `$localize` tag simply returns the original template with
+ * the interpolations evaluated - effectively an identity for the original template literal.
+ *
+ * Generally this version is used in development or where there is no need to transform the
+ * localized template literals.
+ *
+ * @param messageParts a collection of the static parts of the template string.
+ * @param expressions a collection of the values of each placeholder in the template string.
+ * @returns the reconstituted string, with the `messageParts` and `expressions` interleaved
+ * together.
+ */
+
+(function(global: any) {
+  const $localize: Localize = function localize(): string {
+    const args = $localize.translate ? $localize.translate(arguments as any) :
+                                       Array.prototype.slice.apply(arguments);
+    const messageParts = args[0];
+    let message = messageParts[0];
+    for (let i = 1; i < messageParts.length; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < messageParts.length; i++) {
```
If the first args is skipped

Reply:
See https://github.com/angular/angular/pull/31609#discussion_r309057798